### ISSUE TITLE
JohnTravoltageView's accessibleContent

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#176]
The constructor is initialized with accessibleContent: null (probably to block ScreenView's default), and in the next line after the constructor is initialized:

```js
    ScreenView.call( this, {
      // ...
      accessibleContent: null
    } );

    this.accessibleContent = { /* ... */ };
```

First of all, ScreenView's accessibleContent is only a default, so presumably the custom accessibleContent could be inlined in the constructor (correct?)

Second of all, it's not documented why we're using a fully custom version. If this is warranted, it should be documented, although somehow extending or providing different options to ScreenView's default would be preferable.

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
I believe the above commits handles all concerns @jonathanolson, would you mind checking?

> First of all, ScreenView's accessibleContent is only a default, so presumably the custom accessibleContent could be inlined in the constructor (correct?)

True.

> It's not documented why we're using a fully custom version

For a while, designers wanted very different markup for the ScreenViews in BASE and john-travoltage, now they should both look like the content created by https://github.com/phetsims/joist/commit/2c166f692bd4fa4a207d287406a6ed4dd9e0e618.

**Comment by jonathanolson** [comment]
The following:
```js
var self = this;
var visibilityListener = function() {
  self.accessibleHidden = !self.visible;
};
visibilityListener();
this.on( 'visibility', visibilityListener );
this.disposeScreenView = function() {
  this.off( 'visibility', visibilityListener );
};

// ...
dispose: function() {
  Node.prototype.dispose.call( this );
  this.disposeScreenView();
}
```

seems more verbose than:
```js
this.accessibleHiddenListener = this.updateAccessibleHidden.bind( this );
this.on( 'visibility', this.accessibleHiddenListener );
this.updateAccessibleHidden();

// ...
updateAccessibleHidden: function() {
  this.accessibleHidden = !this.visible
},
dispose: function() {
  Node.prototype.dispose.call( this );
  this.off( 'visibility', this.accessibleHiddenListener );
}
```

Also note both examples have the super-call on dispose(), which should be added.

**Comment by jessegreenberg** [comment]
Sounds good, I changed the implementation to the above.

I added ```Node.prototype.dispose.call( this );```, though I am a little surprised it is needed.  Except for the above change, we are using ```Node.prototype.dispose.call( this );``` 0 times throughout the project.  All types that extend Node directly should call this?

**Comment by jonathanolson** [comment]
> All types that extend Node directly should call this?

Yes, IF they override dispose().

I'll create an issue for that in Scenery.

**Comment by jonathanolson** [comment]
Looks good, thanks.
